docs/design/specs/Ecma-335-Augments.md

```diff
@@ -502,8 +502,8 @@ implements. The remaining information specifies the virtual instance or non-virt
 provides the implementation.
 
 While the syntax specified here (as well as the actual metadata format (paragraph II.22.27) allows any virtual
-method to be used to provide an implementation, a conforming program shall provide a virtual instance method
-actually implemented directly on the type or a static method implemented on the type or on a base type.
+method to be used to provide an implementation, a conforming program shall provide a virtual instance
+or static method actually implemented directly on the type.
```

Comment:
Is there anything to change in MethodImpl section? I think we also agreed to specify that the MethodImpl entry must use a MethodDef index to refer to a static "body" method.

Reply:
Clarified in the latest David's commit.